install/k8s/k8s1.4/contiv.yaml

```diff
@@ -15,7 +15,7 @@ data:
   # The CNI network configuration to install on each node.
   cni_config: |-
     {
-      "cniVersion": "0.1.0",
+      "cniVersion": "0.6.0",
```

Comment:
Why is a 1.4 manifest being upgraded to cni 0.6.0? I would expect only k8s 1.9 and greater to use 0.6.0.

Reply:
Thanks for the comment @danehans. Sure, I'll correct it. Btw, we are planning to deprecate k8s `1.4` in `contiv/install`. k8s `1.4` has already been deprecated in `contiv/netplugin`.

Reply:
@danehans I've reverted the CNI version for k8s 1.4 in the latest commit. I'll deprecate/remove k8s 1.4 in contiv/install in a separate PR.